test-clients/src/main/java/com/hedera/services/bdd/suites/contract/ContractCallSuite.java

```diff
@@ -234,94 +228,96 @@ HapiApiSpec fridayThe13thSpec() {
 	HapiApiSpec depositSuccess() {
 		return defaultHapiSpec("DepositSuccess")
 				.given(
-						fileCreate("payableBytecode").path(PATH_TO_PAYABLE_CONTRACT_BYTECODE),
+						fileCreate("payableBytecode").path(ContractResources.PAYABLE_CONTRACT_BYTECODE_PATH),
 						contractCreate("payableContract").bytecode("payableBytecode").adminKey(THRESHOLD)
 				).when(
-						contractCall("payableContract", DEPOSIT, 1_000L)
-								.via("payTxn").sending(1_000L)
+						contractCall("payableContract", ContractResources.DEPOSIT_ABI, depositAmount)
+								.via("payTxn").sending(depositAmount)
 				).then(
 						getTxnRecord("payTxn")
 								.hasPriority(recordWith().contractCallResult(
 										resultWith().logs(inOrder()))));
 	}
 
 	HapiApiSpec multipleDepositSuccess() {
-		return defaultHapiSpec("DepositSuccess")
+		return defaultHapiSpec("MultipleDepositSuccess")
 				.given(
-						fileCreate("payableBytecode").path(PATH_TO_PAYABLE_CONTRACT_BYTECODE),
+						fileCreate("payableBytecode").path(ContractResources.PAYABLE_CONTRACT_BYTECODE_PATH),
 						contractCreate("payableContract").bytecode("payableBytecode").adminKey(THRESHOLD)
 				)
 				.when()
 				.then(
 						withOpContext((spec, opLog) -> {
 							for (int i = 0; i < 10; i++) {
 								var subOp1 = balanceSnapshot("payerBefore", "payableContract");
-								var subOp2 = contractCall("payableContract", DEPOSIT, 1_000L)
-										.via("payTxn").sending(1_000L);
+								var subOp2 = contractCall("payableContract", ContractResources.DEPOSIT_ABI, depositAmount)
+										.via("payTxn").sending(depositAmount);
 								var subOp3 = getAccountBalance("payableContract")
-										.hasTinyBars(changeFromSnapshot("payerBefore", +1_000L));
+										.hasTinyBars(changeFromSnapshot("payerBefore", +depositAmount));
 								CustomSpecAssert.allRunFor(spec, subOp1, subOp2, subOp3);
 							}
 						})
 				);
 	}
 
 	HapiApiSpec depositDeleteSuccess() {
-		return defaultHapiSpec("DepositSuccess")
+		long initBalance = 7890;
+		return defaultHapiSpec("DepositDeleteSuccess")
 				.given(
-						fileCreate("payableBytecode").path(PATH_TO_PAYABLE_CONTRACT_BYTECODE),
+						cryptoCreate("beneficiary").balance(initBalance),
+						fileCreate("payableBytecode").path(ContractResources.PAYABLE_CONTRACT_BYTECODE_PATH),
 						contractCreate("payableContract").bytecode("payableBytecode").adminKey(THRESHOLD)
 				).when(
-						contractCall("payableContract", DEPOSIT, 1_000L)
-								.via("payTxn").sending(1_000L)
+						contractCall("payableContract", ContractResources.DEPOSIT_ABI, depositAmount)
+								.via("payTxn").sending(depositAmount)
 
 				).then(
-						balanceSnapshot("payerBefore", GENESIS),
-						contractDelete("payableContract").transferAccount(GENESIS),
-						getAccountBalance(GENESIS)
-								.hasTinyBars(changeFromSnapshot("payerBefore", +1_000L))
+//						balanceSnapshot("payerBefore", MASTER),
```

Comment:
Should this commented code be removed?

Reply:
Thanks Neeha! Good catch! My fix somehow dismissed your approval, so I request your review once more. Thanks!